cli/man/splinter-circuit-vote.1.md

```diff
@@ -0,0 +1,93 @@
+% SPLINTER-CIRCUIT-VOTE(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**splinter-circuit-vote** — Submits a vote to accept or reject a circuit proposal
+
+SYNOPSIS
+========
+**splinter circuit vote** \[**FLAGS**\] \[**OPTIONS**\] <circuit_id> --accept --reject
+
+DESCRIPTION
+===========
+Vote on a new circuit proposal by specifying the circuit ID of the circuit the node
+is voting on, as well as the flag to `--accept` or `--reject`. The generated ID of
+a proposed circuit can be viewed using the `splinter-circuit-proposals` command and
+this circuit ID is used to specify the circuit proposal being voted on. A circuit
+proposal is viewable, via the `splinter-circuit-proposals` or `splinter-circuit-show`
+commands, by all proposed member nodes unless a proposed member node votes to
+reject the circuit proposal. A circuit proposal needs to be voted on by all proposed
+members that did not propose the circuit in the first place. Circuit proposers have
+an assumed `ACCEPT` vote, as these nodes requested the creation of the circuit.
+
+FLAGS
+=====
+`--accept`
+: Accept the circuit proposal specified.
+
+`-h`, `--help`
+: Prints help information.
+
+`-q`, `--quiet`
+: Decrease verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`--reject`
+: Reject the circuit proposal specified.
+
+`-V`, `--version`
+: Prints version information.
+
+`-v`
+: Increases verbosity (the opposite of -q). Specify multiple times for more
+  output.
+
+OPTIONS
+=======
+`-k`, `--key` <private-key-file>
+: Specifies the full path to the private key file.
+
+`-U`, `--url` <url>
+: Specifies the URL for the `splinterd` REST API. The URL is required unless
+  `$SPLINTER_REST_API_URL_ENV` is set.
+
+ARGUMENTS
+=========
+`<circuit-id>`
+: Specify the circuit ID of the circuit to be voted on.
+
+EXAMPLES
+========
+* The proposed circuit has ID `1234-ABCDE`.
+
+The following command displays a member node voting to accept the circuit proposal:
+```
+$ splinter circuit vote \
+  --key <proposed-member-node-private-key> \
```

Comment:
All examples should try and avoid shell characters such as "<" and ">" because a user cut-n-pasting these will have a bad day.

Reply:
The intention is that the user would replace `<...>` with their own private key, so what would you suggest we use instead of the <> characters to display that? I think these characters are used pretty widely across the man pages where it is necessary

Reply:
Possible approaches that I've used in the past:
- Curly braces instead of angle brackets. They're still special chars, but they have less horrible effects when used.

- No brackety characters, but put the replaceable item in ALL-CAPS to make it extremely obvious. 

Best practice is to use a short string for the replaceable item, then explain the details in the text above. For example, "Replace MEMBER-KEY with  the private key of the proposed member node."